controller_report_xls/controllers/main.py

```diff
@@ -1,45 +1,174 @@
-# coding: utf-8
+# -*- coding: utf-8 -*-
+
+from ..controllers.xfstyle import css2excel
 
 from openerp.addons.report.controllers import main
 from openerp.addons.web.http import route, request  # pylint: disable=F0401
 from werkzeug import url_decode  # pylint: disable=E0611
 import simplejson
-import lxml.html
+from lxml import etree
+import cssutils
+from cssutils import parseString
```

Comment:
Can you import this inside a try please.

As it is not a normal dependency declare this repository bring unexpected traceback with just run the server with this depclared.

more information:

https://yelizariev.github.io/odoo/development/2015/07/06/external-dependencies.html